package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev -p 8888",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "netlify-build": "npm run build"
```

Comment:
Why it'll have to add another duplicated script? Why not just use `npm run build`?